packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.ts

```diff
@@ -39,7 +40,7 @@ const tokenDetails: {
 };
 
 class CardPayDepositWorkflowTransactionAmountComponent extends Component<CardPayDepositWorkflowTransactionAmountComponentArgs> {
-  @tracked amount = 0;
+  @tracked amount = '0';
```

Comment:
We're storing the amount as a string and validating it before we update it when an `input` event comes in, to prevent floating point issues. We convert it to a `FixedNumber` when we want to do any operations on it.

Reply:
(Will definitely need more validation)